tests/manual-test-cases/Group16-WindowsClient/16-1-Windows-VIC-Machine.robot

```diff
@@ -18,7 +18,7 @@ Resource  ../../resources/Util.robot
 Test Setup  Set Test Environment Variables
 
 *** Variables ***
-${ver}  0.6.0
+${ver}  8351
```

Comment:
just a severely outdated test that needs love in a different PR.  I used the most recent one that we have in another test, I believe 8351 was 0.8 ultimately.